config/config_helpers.go

```diff
@@ -208,6 +211,30 @@ func parseGetEnvParameters(parameters string) (EnvVar, error) {
 	return envVariable, nil
 }
 
+// runCommand is a helper function that runs a command and returns the stdout as the interporation
+// result
+func runCommand(parameters string, terragruntOptions *options.TerragruntOptions) (string, error) {
+	args := parseParamList(parameters)
+
+	if len(args) == 0 {
+		return "", errors.WithStackTrace(EmptyStringNotAllowed("parameter to the run_cmd function"))
+	}
+
+	currentPath := filepath.Dir(terragruntOptions.TerragruntConfigPath)
+
+	command := &exec.Cmd{
+		Path: args[0],
+		Args: args,
+		Dir:  currentPath,
+	}
+	stdout, err := command.Output()
```

Comment:
We can't use this function since it currently forces us to use `terragruntOptions.WorkingDir`. There are two options:

A - We change the function to accept a custom `workingDir`.
B - We don't use it and just posts the stdout to the logs

I'm leaning towards B. I think that most use cases for `run_cmd` will be running small shell scripts that take less than a second to execute, so I think that streaming the logs is overkill, we can simply add the stdout to the logs once the function finishes its execution.

thoughts?

Reply:
I would vote for A. Should be a minor change and we improve the user experience. 

I can see this feature being used by teams to run `terragrunt output xxx` in some other folder to get outputs from other modules. Having streaming logs for that sort of thing is quite handy.

Thanks!

Reply:
ok, A it is

Reply:
take a look now